Formula/textidote.rb

```diff
@@ -0,0 +1,55 @@
+class Textidote < Formula
+  desc "Spelling, grammar and style checking on LaTeX documents"
+  homepage "https://sylvainhalle.github.io/textidote"
+  url "https://github.com/sylvainhalle/textidote/archive/refs/tags/v0.8.3.tar.gz"
+  sha256 "8c55d6f6f35d51fb5b84e7dcc86a4041e06b3f92d6a919023dc332ba2effd584"
+  license "GPL-3.0-only"
```

Comment:
```suggestion
  license "GPL-3.0-or-later"
```

license ref, https://github.com/sylvainhalle/textidote/blob/ab9de53c1fe7a04826a98850c75d3f4fb6a0811e/Source/Core/src/ca/uqac/lif/textidote/RuleException.java#L1-L17